pkg/network/driver/common.go

```diff
@@ -402,7 +403,7 @@ func (h *NetworkUtilsHandler) BindTapDeviceToBridge(tapName string, bridgeName s
 }
 
 func (h *NetworkUtilsHandler) DisableTXOffloadChecksum(ifaceName string) error {
-	if err := EthtoolTXOff(ifaceName); err != nil {
+	if err := ethtool.EthtoolTXOff(ifaceName); err != nil {
```

Comment:
Why not use the now created `TXChecksumOff`?

Reply:
This is just an intermediate step, `DisableTXOffloadChecksum` is removed eventually.